### PR TITLE
13 adding playlist recommendation

### LINKED ISSUE #13: Adding playlist recommendation
Adding the playlist recommendation based on the weather using the Spotify Api

### PR DESCRIPTION
**Pull request by miador**
Resolves #13

### CODE REVIEW

**Comment by grace-omotoso** [comment]
@shivam @trinhcaokhoa please, review code. @norachams @Bhawna1203  please test functionality.

**Comment by kph4m** [comment]
Looks great! Is it possible to compile a playlist of songs rather than just 1?

**Comment by norachams** [comment]
> @Shivam @trinhcaokhoa please, review code. @norachams @Bhawna1203 please test functionality.

Works perfectly. I think we should add a playlist of songs instead of just 1 like Khiem mentioned.

File: src/App.js [hunk 1]
```diff
@@ -8,59 +8,67 @@ function App() {
...
-						setResults(result);
-					}
-				},
-				(error) => {
-					setIsLoaded(true);
-					setError(error);
-				},
-			);
-	}, [city]);
+  let weatherConditions = new Map([
```
Comment (grace-omotoso) [inline src/App.js]: I think we should add sunny weather too. I see you already added songs for common weather conditions. I am not sure what the spotify API offers but can we explore more options as we currently have with the items to bring functionality.?
Also, as mentioned by others, does the API allow us to pick from a list of playlist? However, we will work based on the functionalities of the API

**Comment by miador** [comment]
Yeah, we can do that. For sunny weather there's also a playlist i included. If you have specific playlist suggestions, let me know.

File: src/App.js (hunk 1 above)
Comment (miador) [reply src/App.js]: In weather API, there's no sunny condition as far as i saw. They included it as "Clear".

**Comment by trinhcaokhoa** [comment]
> @Shivam @trinhcaokhoa please, review code. @norachams @Bhawna1203 please test functionality.

I suggest creating a CSS file for the Spotify div and then importing it in App.css or directly importing it into App.js. It's quite not 'clean' to me when you modify the style directly in HTML

**Comment by miador** [comment]
> > @Shivam @trinhcaokhoa please, review code. @norachams @Bhawna1203 please test functionality.
> 
> I suggest creating a CSS file for the Spotify div and then importing it in App.css or directly importing it into App.js. It's quite not 'clean' to me when you modify the style directly in HTML

That iframe is autogenerated by spotify itself. I am not sure if we can load style from css for the iframe. If so, i would need some help to do it.

**Comment by miador** [comment]
image

**Comment by miador** [comment]
There are some changes in unrelated files. That's because prettier formatting.

**Comment by trinhcaokhoa** [comment]
> > > @Shivam @trinhcaokhoa please, review code. @norachams @Bhawna1203 please test functionality.
> > 
> > 
> > I suggest creating a CSS file for the Spotify div and then importing it in App.css or directly importing it into App.js. It's quite not 'clean' to me when you modify the style directly in HTML
> 
> That iframe is autogenerated by spotify itself. I am not sure if we can load style from css for the iframe. If so, i would need some help to do it.

I got it. If so, the code look good to me.

**Comment by indevi** [comment]
LGTM

**Comment by miador** [comment]
@youngjun827 please test functionality

**Comment by youngjun827** [comment]
AWESOME JOB GUYS! I can confirm everything works.

**Comment by pixawna** [comment]
This branch has some conflicts could you please check it out :)

**Review by grace-omotoso — APPROVED** [review]

**Pull request merged**